src/main/java/com/klarna/hiverunner/HiveServerContainer.java

```diff
@@ -132,16 +132,6 @@ public TemporaryFolder getBaseDir() {
     }
 
     /**
-     * Executes a hive script.
-     * @param hiveql hive script statements.
-     */
-    public void executeScript(String hiveql) {
```

Comment:
Import statements can be used in statements and scripts. Both `CliDriver` and `BeeLine` will first read a script file in, split its lines into statements and then execute them individually. This way they can support their custom commands.